solidity/test/bridge/Bridge.Deployment.test.ts

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+import { deployments, ethers, upgrades } from "hardhat"
+import chai, { expect } from "chai"
+import chaiAsPromised from "chai-as-promised"
+
+import type { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
+import type { ProxyAdmin, Bridge } from "../../typechain"
+import type { TransparentUpgradeableProxy } from "../../typechain/TransparentUpgradeableProxy"
+
+chai.use(chaiAsPromised)
+
+const { AddressZero } = ethers.constants
+
+describe("Bridge - Deployment", async () => {
+  let deployer: SignerWithAddress
+  let governance: SignerWithAddress
+  let keepTechnicalWalletTeam: SignerWithAddress
```

Comment:
Let's make it `esdm` please.

Reply:
3ac5b1021e381b2e361e40bc4e0c13ad70a03633